geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStoreFactory.scala

```diff
@@ -255,4 +275,5 @@ object AccumuloDataStoreFactory extends GeoMesaDataStoreInfo {
       caching: Boolean
     ) extends DataStoreQueryConfig
 
+  case class RemoteScansEnabled(arrow: Boolean, bin: Boolean, density: Boolean, stats: Boolean)
```

Comment:
Hmm...  Should this factored out?  Meh, probably not?  Maybe later?

Reply:
maybe when we get to 3 of em ;)